addons/crm/models/crm_lead.py

```diff
@@ -222,7 +222,9 @@ class Lead(models.Model):
     # Statistics
     meeting_count = fields.Integer('# Meetings', compute='_compute_meeting_count')
     # UX
-    ribbon_message = fields.Char('Ribbon message', compute='_compute_ribbon_message')
+    ribbon_message = fields.Char('Ribbon message')
+    will_write_email = fields.Boolean('Will the customer email will be overwritten?', compute='_compute_will_write_email')
```

Comment:
Maybe rename to partner_email_update ? or something like that.

I think we could also have a shorter string, like "Partner Email to Update"